gomruby.h

```diff
@@ -184,4 +185,35 @@ static inline struct RClass *_go_mrb_class_ptr(mrb_value o) {
   return mrb_class_ptr(o);
 }
 
+static inline void _go_set_gc(mrb_state *m, int val) {
+  mrb_gc *gc = &m->gc;
+  gc->disabled = val;
+}
+
+static inline void _go_disable_gc(mrb_state *m) {
+  _go_set_gc(m, 1);
+}
+
+static inline void _go_enable_gc(mrb_state *m) {
+  _go_set_gc(m, 0);
+}
+
+static inline int _go_isdead(mrb_state *m, mrb_value o) {
```

Comment:
Can we add comments within this function to explain what each conditional is? I used to know but no longer know and think it'd be helpful. :)